src/components/files/FILE_MIME_TYPES.ts

```diff
@@ -0,0 +1,54 @@
+export const IMAGE_TYPES = [
```

Comment:
Yes, but it doesn't distinguish between PDF and Word files, and it doesn't recognize some other types, so a lot of stuff comes out as "Other". It would definitely be simpler in code, but less nice for users

Reply:
Hmm the idea of that was to avoid this. I think we have two options here - either update category logic to be useful for us here or just remove it. On one hand it would be nice to have this list handled for us and not have to bundle it. On the other, maybe the API shouldn't concern itself with that. What do you think?

Reply:
If this is the point of the `category` field, I think I'd be in favor of just updating the API logic. I don't **think** it would become a maintenance headache... there's only so many common file types that we might expect users to upload, right? 😬

Reply:
Haha one would hope that. Can you update that API code then?

Reply:
Sure, no problem